talk/owt/sdk/conference/conferenceclient.cc

```diff
@@ -157,10 +162,12 @@ void ConferenceInfo::TriggerOnStreamMuteOrUnmute(
     }
   }
 }
+
 enum ConferenceClient::StreamType : int {
   kStreamTypeCamera = 1,
   kStreamTypeScreen,
   kStreamTypeMix,
+  kStreamTypeQuic,
```

Comment:
`kStreamTypeData`? The stream types listed here seem to describe their content, rather than their protocols.

Reply:
done.